docker/configure_workers_and_start.py

```diff
@@ -78,85 +82,83 @@
 MAIN_PROCESS_UNIX_SOCKET_PUBLIC_PATH = "/run/main_public.sock"
 MAIN_PROCESS_UNIX_SOCKET_PRIVATE_PATH = "/run/main_private.sock"
 
-# A simple name used as a placeholder in the WORKERS_CONFIG below. This will be replaced
-# during processing with the name of the worker.
-WORKER_PLACEHOLDER_NAME = "placeholder_name"
+# We place a file at this path to indicate that the script has already been
+# run and should not be run again.
+MARKER_FILE_PATH = "/conf/workers_have_been_configured"
+
+
+@dataclass
+class WorkerTemplate:
```

Comment:
I'm a bit confused at why this is a "template" -- it seems to be used both as a template and a concrete config?

Reply:
will add a comment, but other than merging multiple templates together (which is a little bit dodgy, sure), it is suppose to be independent of e.g. the worker name.